### PR TITLE
Added Car dto

### PR DESCRIPTION
**Pull request by ArtemReizer**

### CODE REVIEW

**Review by CodeWebWeaver — DISMISSED** [review]

File: src/main/java/com/jvavateam/carsharingapp/dto/car/CarDtoRequest.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.jvavateam.carsharingapp.dto.car;
+
+import com.jvavateam.carsharingapp.model.Car;
+import java.math.BigDecimal;
+
+public record CarDtoRequest(String model, String brand, int inventory,
```
Comment (holsoni) [inline src/main/java/com/jvavateam/carsharingapp/dto/car/CarDtoRequest.java]: Add validation to fields

**Review by holsoni — CHANGES_REQUESTED** [review]

File: src/main/java/com/jvavateam/carsharingapp/dto/car/CarDtoRequest.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.jvavateam.carsharingapp.dto.car;
+
+import com.jvavateam.carsharingapp.model.Car;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+public record CarDtoRequest(@NotNull String model,
+                            @NotNull String brand,
+                            @Min(1) int inventory,
```
Comment (Dimagaa) [inline src/main/java/com/jvavateam/carsharingapp/dto/car/CarDtoRequest.java]:
```suggestion
                            @Positive int inventory,
```

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 3]
```diff
@@ -1 +1,3 @@
-
+databaseChangeLog:
+  - include:
+      file: db/changelog/changes/02.10.CreateCarsTable.yaml
```
Comment (Dimagaa) [inline src/main/resources/db/changelog/db.changelog-master.yaml]:
```suggestion
      file: db/changelog/changes/02.10.2023.CreateCarsTable.yaml
```

**Review by Dimagaa — DISMISSED** [review]
Good Job! Just few suggestions

**Review by holsoni — APPROVED** [review]

**Review by Dimagaa — APPROVED** [review]

**Pull request merged**